Example/Example.js

```diff
@@ -59,7 +60,7 @@ const getSceneStyle = () => ({
 });
 const BackgroundView = () => <View style={{ position: 'absolute', top: 0, bottom: 0, left: 0, right: 0, backgroundColor:'red' }}/>
 const Example = () => (
-  <Router createReducer={reducerCreate} tintColor="red" getSceneStyle={getSceneStyle}>
+  <Router engine={MobXEngine} createReducer={reducerCreate} tintColor="red" getSceneStyle={getSceneStyle}>
```

Comment:
Ideally we could get away with the end users additional complexity ending right here. We will see when the `navigationStore` changes are in.

Reply:
I was successful in making this the most complex part of this! YAY!